hack/start-api

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source $(dirname "${0}")/common
+
+case $(k8s_env) in
+    $DOCKER_FOR_DESKTOP)
+        echo "Found Docker Kubernetes ..."
+        LOCAL_K8S_HOST_IP=localhost
+        LOCAL_K8S_PORT=6443
+        go run cmd/gardener-apiserver/main.go \
+			--authentication-kubeconfig ~/.kube/config \
+			--authorization-kubeconfig ~/.kube/config \
+			--etcd-servers=http://$LOCAL_K8S_HOST_IP:$LOCAL_K8S_PORT \
```

Comment:
`LOCAL_K8S_PORT` is `6443`, but it is used in the `--etcd-servers` flag.
Actually, the etcd servers should be configured like this: `--etcd-servers=http://localhost:32379`:

```
$ curl localhost:32379
404 page not found
```

With this setup, Gardener API server does not have connectivity to etcd.

Reply:
my bad. switched the etcd port back to 32379